test/functional/forms/textareas.js

```diff
@@ -0,0 +1,31 @@
+describe('forms-textarea', () => {
```

Comment:
Please test the 3 variants independently, otherwise a11y tests will only target the first textarea. I'm not sure the disabled state is accessible...